docs: update information about tsdb usage in cluster version (#4678)
VictoriaMetrics/VictoriaMetrics: README.md

Cardinality explorer is built on top of [/api/v1/status/tsdb](#tsdb-stats).

> If you are using [cluster version of VictoriaMetrics](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html) each 
> vmstorage tracks the number of stored timeseries individually. vmselect requests stats via [/api/v1/status/tsdb](#tsdb-stats) 
> from each vmstorage node and merges the results. This may lead to inflated values when samples for the same time series 
> are spread across multiple vmstorage nodes due to [replication](#replication) or rerouting.
> It happens because vmstorage fetches the collected tsdb stats from every vmstorage node and merges it.
> The `seriesCountByMetricName` stats is just summed across vmstorage nodes.
> 

Reviewer: ```suggestion
In [cluster version of VictoriaMetrics](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html) each vmstorage tracks the stored time series individually. 
vmselect requests stats via [/api/v1/status/tsdb](#tsdb-stats) API from each vmstorage node 
and merges the results by summing per-series stats. 
This may lead to inflated values when samples for the same 
time series are spread across multiple vmstorage nodes due to [replication](#replication) or [rerouting](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html?highlight=rerouting#cluster-availability).
```
